eng/common/pipelines/templates/steps/eng-common-workflow-enforcer.yml

```diff
@@ -1,18 +1,21 @@
+# cSpell:ignore changedfiles
+# cSpell:ignore Committish
+
 steps:
+  - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
+    parameters:
+      DefaultBranchVariableName: DefaultBranch
```

Comment:
Isn't this the default value of the template? Do we need to explicitly pass it?

Reply:
The $(DefaultBranch) used in next step has a reference. Otherwise people maybe confused where the value comes from

Reply:
I agreed to remove and add comment on where the variable comes from. In case people make changes to the var which the step following miss the var.

Reply:
Done